src/snek5000/assets/SIZE.j2

```diff
@@ -42,15 +42,17 @@ c
       {{ parameter("lpert", params.oper.max.perturb) }}  ! max number of perturbations
       {{ parameter("toteq", params.oper.max.scalars_cons) }}  ! max number of conserved scalars in CMT
       {{ parameter("nsessmax", params.oper.max.sessions) }}  ! max sessions to NEKNEK
-      {{ parameter("lxo", params.oper.elem.order_out) }}  ! max GLL points on output (lxo>=lx1)
+      {{ parameter("lxo", order_out) }}  ! max GLL points on output (lxo>=lx1)
       {{ parameter("mxprev", params.oper.max.dim_proj) | trim }}
       {{ parameter("lgmres", params.oper.max.dim_krylov) }}  ! max dim of projection & Krylov space
       {{ parameter("lorder", params.oper.max.order_time) }}  ! max order in time
       {{ parameter("lx1m", order_mesh_solver) }}  ! GLL points mesh solver
       {{ parameter("lfdm", params.oper.misc.fast_diag | int) }}  ! set to 1 for fast diagonalization method
-      {{ parameter("lelx", max_nx) | trim }}
```

Comment:
Isn't it strange to remove this `| trim` just for `max_nx`?

Reply:
See the change below... Earlier the comment was written next to `max_nz`. Now I moved it up because `max_nz` is pointless for the 2D case

Reply:
What I mean is that there is now `{{ parameter("lelx", max_nx) }}` and `{{ parameter("lely", max_ny) | trim }}`...

Reply:
It is alright. The `trim` filter in Jinja2 simply removes whitespace.